go.mod

```diff
@@ -1,16 +1,17 @@
 module storj.io/linksharing
 
-go 1.14
+go 1.13
```

Comment:
:thinking: shouldn't we keep it in 1.14?

Reply:
Definitely not. Neither of the packages requires it and thus we should not force everyone to update their go version.